.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
```

Comment:
- [ ] Please make sure to write a good description in your pull request telling exactly what you have done in this pull request
![image](https://user-images.githubusercontent.com/74744787/184949712-99a6a397-11f5-4442-a3b7-dedaa08c71e3.png)